contents/docs/user-guides/group-analytics.md

```diff
@@ -71,21 +71,21 @@ posthog.init('[your api key]', {
 });
 ```
 
-Subsequent calls to `posthog.group()` with the same group type but a different group key make the new group be active.
+Subsequent calls to `posthog.group()` with the same group type but a different group key make the new Group active.
 
 #### Handling logging out
 
-When the user logs out it's important to call `posthog.reset()` to avoid new events being registered under registered groups and the logged in user.
+When the user logs out, it's important to call `posthog.reset()` to avoid new events being registered under registered groups and the logged-in user.
 
 ### [posthog-python](https://posthog.com/docs/integrate/server/python)
 
 Update to version 1.4.3 or above to make use of the new functionality.
 
 ```python
-# Capturing an event with groups
+# Capturing an event with Groups
```

Comment:
The capitalization of every instance of the word Group has me confused. What's the intended goal?